libparsec/crates/client/tests/unit/client/list_frozen_users.rs

```diff
@@ -0,0 +1,24 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+use libparsec_tests_fixtures::prelude::*;
+
+use super::utils::client_factory;
+use crate::ClientListFrozenUsersError;
+
+#[parsec_test(testbed = "coolorg", with_server)]
+async fn ok(env: &TestbedEnv) {
```

Comment:
I would slip the test into 2 part, one is the happy path with alice and the other is the unhappy path with the `AutorNotAllowed` error using mallory